website/src/pages/landing.css

```
@@ -0,0 +1,790 @@
+
+  --lp-container: 1200px;
+  --lp-gutter: 32px;
+  --lp-nav-h: 56px;
+
+  font-family: var(--lp-f-body);
+  font-size: 16px;
+  line-height: 1.55;
+  color: var(--lp-fg-1);
+  background: var(--lp-bg-0);
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.9.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)